utils/bccmd/Makefile

```diff
@@ -0,0 +1,44 @@
+# SPDX-Identifier-License: GPL-2.0-or-later
+#
+# Copyright (C) 2023 Vincent Tremblay <vincent@vtremblay.dev>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bccmd
+PKG_VERSION:=6.0
+PKG_RELEASE:=1
+
+PKG_MAINTAINER=Vincent Tremblay <vincent@vtremblay.dev>
+
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_FIXUP:=autoreconf
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/bccmd/bccmd.git
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_MIRROR_HASH:=2cd7e6eb755785101a60e1da70de2d7788ba7bd2459b576e82b5851ed422588e
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/bccmd
+	TITLE:=CSR BlueCore command interface
+	SECTION:=utils
+	CATEGORY:=Utilities
+	URL:=https://github.com/bccmd/bccmd.git
+	DEPENDS:=+bluez-libs
```

Comment:
```suggestion
  SECTION:=utils
  CATEGORY:=Utilities
  TITLE:=CSR BlueCore command interface
  URL:=https://github.com/bccmd/bccmd.git
  DEPENDS:=+bluez-libs
```

use 2 \<space\> instead of \<tab\>

Reply:
I'm not too sure about this one. From the spec, Makefiles requires tabs and not spaces. 
What is the reason to change it with spaces? (I really do want to understand!)

Reply:
> https://stackoverflow.com/questions/28712585/when-to-use-space-or-tab-in-makefile
> In order for make to tell the difference between a recipe and things that are not a recipe, it uses TAB characters. So, lines that begin with TAB are assumed to be part of a recipe (so they are shell scripts and passed to the shell for parsing), and lines that do not begin with TAB cannot be part of a recipe (so they cannot be shell scripts: they must be make syntax).

Tabs also works here, but let's keep constant with other makefiles in tree.

Reply:
thanks for the explanation, I really appreciate. 
It's fixed!